baseline/pytorch/seq2seq/train.py

```diff
@@ -106,6 +118,7 @@ def _evaluate(self, es, reporting_fns, **kwargs):
             preds.extend(convert_seq2seq_preds(pred, self.tgt_rlut))
             golds.extend(convert_seq2seq_golds(tgt, tgt_lens, self.tgt_rlut))
         metrics = {'bleu': bleu(preds, golds, self.bleu_n_grams)[0]}
+        metrics['acc'] = self._acc(preds, golds)
```

Comment:
if tgt_lens is one shorter in train and eval, the value passed into convert_seq2seq_golds is probably off by one right?

Reply:
it turns out to be right: in `evaluate()` both `tgt` and `tgt_len` are from `batch_dict`, so the length matches the tgt; in `test()`, both of them from the `input_` (after `make_input`) so they also match. the only difference between the two cases is an `<GO>` at the beginning, which is taken care of inside `convert_seq2seq_golds()`.